01-messaging.md

```diff
@@ -37,12 +37,13 @@ The `type` field indicates how to interpret the `payload` field.
 The format for each individual type is specified in a specification in this repository.
 The type follows the _it's ok to be odd_ rule, so nodes MAY send odd-numbered types without ascertaining that the recipient understands it.
 
-A node MUST NOT:
- - send an evenly-typed message not listed here without prior negotiation.
-
-A node MUST:
- - ignore a received message of unknown type, if that type is odd.
- - fail the channels if it receives a message of unknown type, if that type is even.
+A node:
+- MUST NOT send an evenly-typed message not listed here without prior negotiation.
+- when receives message of unknown type
+ - if that type is odd
+   - MUST ignore the received message.
+ - otherwise type is even
+   - MUST fail the channels.
 
```

Comment:
I prefer to separate sending and receiving requirements.  As an implementer, you implement one of them at a time.

So, when implementing "receive_foo" you look through the "A node receiving foo..." (and, for c-lightning, we include and check quotes from the spec, making it especially powerful).

Reply:
Alright, I can go through and group requirements according to sender/receiver for sure.